go/pkg/client/cas.go

```diff
@@ -27,8 +27,7 @@ import (
 
 // UploadIfMissingStats reports the CAS hits/misses.
 type UploadIfMissingStats struct {
-	Hits   map[digest.Digest]struct{}
-	Misses map[digest.Digest]struct{}
+	Misses []digest.Digest
```

Comment:
no need to use struct now?

Reply:
Hmm, actually I'd like to keep this struct, for the ease of extending it (We've had a few times where the args kept piling up, so eventually we made a struct for them). Maybe rename it to `UploadIfMissingResult`?

Reply:
I see. But I prefer to be lazy for using struct here, because I'm not so sure whether we want to have other field sometime soon.

Reply:
OK, maybe we can wait for foundry-x's opinion?

Reply:
I see.

Reply:
I would prefer we not add a struct until we need to. This makes the function signature simpler, similar to MissingBlobs and basic Go functions like Write returning number of bytes written.

Reply:
OK, done.